docs/includes/auth/AuthUser.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Models;
+
+use Jenssegers\Mongodb\Auth\User as Authenticatable;
+
+class User extends Authenticatable
```

Comment:
The developers don't have to create this class in their projects, it is available in the package: https://github.com/mongodb/laravel-mongodb/blob/4.6/src/Auth/User.php

Reply:
I mention on the page that this class already exists, but they need to replace `use Illuminate\Foundation\Auth\User as Authenticatable` with `use Jenssegers\Mongodb\Auth\User as Authenticatable` - is that correct?

Reply:
Oh, sorry, I got confused. Indeed, they do need to create a `User` class that extends the one in our package. 
However, it is `MongoDB\Laravel\Auth\User` that should be used. `Jensseger` corresponds to the old namespace before we took over the package.

Reply:
Ah got it, fixed!